### ISSUE TITLE
 Make more clear about readme parser warnings

### ISSUE THREAD

**Issue by ernilambar** [10up/plugin-check#394]
If there are warnings returned from the Parser, we display message like this:

```
The following readme parser warnings were detected: tested_header_ignored, requires_php_header_ignored
```

Message here is not quite clear. May be we could display more clearer messages for each warnings.

Existing warnings in the Parser:

* contributor_ignored
* requires_php_header_ignored
* tested_header_ignored
* requires_header_ignored
